pkg/networkmanager/networking.css

```diff
@@ -58,8 +64,12 @@
     height: 120px;
 }
 
+#networking-interfaces {
+    min-height: 300px;
```

Comment:
This is just a hack to show all of the new drop down, right?  We need to make that work without this hack, of course.  Just using a standard select with class "ct-select" should probably work.  (The rest of the Networking page has the same problem, it's ooooold....) An example would be the `#services-dropdown` in `pkg/system/services.html`.